project_settings.yml

```diff
@@ -1,8 +1,8 @@
 default:
 
-  project_name: new_config_test
+  project_name: name_of_the_project_folder
 
   project_internal:
     twodii_internal: FALSE
```

Comment:
I think this should be TRUE as a default @jdhoffa

Reply:
although I am seeing it used to be FALSE.. it's the same to me really. Rest looks good to me

Reply:
I went back and forth on this one haha... 
`TRUE` is a better default when we are using it, but `FALSE` is a better default for when externals (EIOPA etc) are using it, and i tend to feel like we should make it easier for them...

Reply:
@Clare2D thoughts?